src/makeCalculator.js

```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(num) {
+      this.result += num;
+    },
+    subtract(num) {
+      this.result -= num;
+    },
+    multiply(num) {
+      this.result *= num;
+    },
+    divide(num) {
+      if (num !== 0) {
+        this.result /= num;
+      } else {
+        throw new Error('Cannot divide by zero!');
+      }
```

Comment:
[CODE KNOWLEDGE] - While it is good to have a check for division by zero, the task does not require throwing an error. Instead, you could consider handling the division by zero case differently, perhaps by setting the result to `null` or some other indicative value, or simply not performing the operation. Remember, the calculator should not throw errors for invalid operations but handle them gracefully.